content/docs/command-reference/checkout.md

```diff
@@ -33,15 +34,19 @@ for more details.
 
 The execution of `dvc checkout` does the following:
 
-- Scans the `.dvc` and `dvc.lock` files to compare against the data files or
-  directories in the <abbr>workspace</abbr>. DVC knows which data
-  (<abbr>outputs</abbr>) match because the corresponding hash values are saved
-  in the `outs` fields in those files. Scanning is limited to the given
+- Scans all `dvc.lock` and `.dvc` files to compare the hash values saved in
```

Comment:
what was wrong with the previous version?

Reply:
It's not incorrect, but it can be easier to read. Specifically:
- Order of files to compare
- It's a full line longer than the new one

Should I roll it back as well?

Reply:
> Note this has been changed a little in 5e29b0d5.